apps/meteor/tests/end-to-end/api/livechat/24-routing.ts

```diff
@@ -91,9 +126,20 @@ import { IS_EE } from '../../../e2e/config/constants';
 			expect(roomInfo.servedBy).to.be.an('object');
 			expect(roomInfo.servedBy?._id).to.not.be.equal(testUser2.user._id);
 		});
+		(IS_EE ? it : it.skip)('should route to contact manager if it is online', async () => {
```

Comment:
I think we should also add the information that `Livechat_assign_new_conversation_to_bot` is true, right?